lib/deliver/loader.rb

```diff
@@ -0,0 +1,13 @@
+require 'fastlane_core/languages'
+
+module Deliver
+  module Loader
+    ALL_LANGUAGES = FastlaneCore::Languages::ALL_LANGUAGES.map(&:downcase).freeze
```

Comment:
I assume your intention with the case where `File::FNM_CASEFOLD` was being used that this should ignore case? If so, I did that by using this array instead.